diff --git a/node_modules/@gnosis.pm/safe-core-sdk/dist/src/managers/contractManager.js b/node_modules/@gnosis.pm/safe-core-sdk/dist/src/managers/contractManager.js
index 0a2f22e..d9a433a 100644
--- a/node_modules/@gnosis.pm/safe-core-sdk/dist/src/managers/contractManager.js
+++ b/node_modules/@gnosis.pm/safe-core-sdk/dist/src/managers/contractManager.js
@@ -76,10 +76,10 @@ class ContractManager {
             customContractAddress: safeAddress,
             customContractAbi: customContracts === null || customContracts === void 0 ? void 0 : customContracts.safeMasterCopyAbi
         });
-        const isContractDeployed = await ethAdapter.isContractDeployed(temporarySafeContract.getAddress());
-        if (!isContractDeployed) {
-            throw new Error('Safe Proxy contract is not deployed in the current network');
-        }
+        // const isContractDeployed = await ethAdapter.isContractDeployed(temporarySafeContract.getAddress());
+        // if (!isContractDeployed) {
+        //     throw new Error('Safe Proxy contract is not deployed in the current network');
+        // }
         return temporarySafeContract;
     }
     async getMultiSendContract({ ethAdapter, safeVersion, chainId, customContracts }) {
@@ -91,10 +91,10 @@ class ContractManager {
             customContractAddress: customContracts === null || customContracts === void 0 ? void 0 : customContracts.multiSendAddress,
             customContractAbi: customContracts === null || customContracts === void 0 ? void 0 : customContracts.multiSendAbi
         });
-        const isContractDeployed = await ethAdapter.isContractDeployed(multiSendContract.getAddress());
-        if (!isContractDeployed) {
-            throw new Error('Multi Send contract is not deployed in the current network');
-        }
+        // const isContractDeployed = await ethAdapter.isContractDeployed(multiSendContract.getAddress());
+        // if (!isContractDeployed) {
+        //     throw new Error('Multi Send contract is not deployed in the current network');
+        // }
         return multiSendContract;
     }
 }
